taskvine/src/manager/vine_manager.c

```diff
@@ -4862,6 +4879,8 @@ int vine_set_category_mode(struct vine_manager *q, const char *category, vine_ca
 		case CATEGORY_ALLOCATION_MODE_MAX:
 		case CATEGORY_ALLOCATION_MODE_MIN_WASTE:
 		case CATEGORY_ALLOCATION_MODE_MAX_THROUGHPUT:
+		case CATEGORY_ALLOCATION_MODE_GREEDY_BUCKETING:
+		case CATEGORY_ALLOCATION_MODE_EXHAUSTIVE_BUCKETING:
```

Comment:
When to use `BUCKETING_` and when to use `CATEGORY_`?

Reply:
BUCKETING_ is internal to bucketing code (decide how to recompute bucketing state), CATEGORY_ is used to connect bucketing code to category code.